test/data/queries.graphql

```diff
@@ -275,6 +286,9 @@ fragment investigationFields on Investigation {
   source
   owner
   groups
+  object_ids
+  investigated_observables
+  targets { ...identitySpecificationFields }
```

Comment:
It's very hard to identify typos in this big graphql query, I mean the query.graphql file. Like we did for the new entities, I would move investigation queries to a new file.

Reply:
Thanks. I added this file, but I replicated all the fragments because I could not reuse `fragments.graphql` which contains `feedbackFields` fragment that is not used in investigation queries.